spec/requests/api/v1/topics/index_spec.rb

```diff
@@ -3,40 +3,62 @@
 describe 'GET api/v1/topics', type: :request do
   let(:user) { create(:user) }
   let!(:headers) { auth_headers }
-  let!(:topics) { create_list(:topic, 3) }
 
   subject { get api_v1_topics_path, headers: headers, as: :json }
-  before { subject }
 
   context 'when there are topics' do
+    let!(:topics) { create_list(:topic, 3) }
+
     it 'returns a successful response' do
+      subject
+
       expect(response).to have_http_status(:success)
     end
 
     it 'returns the correct amount of topics' do
+      subject
+
       expect(json[:topics].size).to eq(3)
     end
 
     it 'returns all the topics' do
       subject
+
       expect(json[:topics].pluck(:id, :name)).to match_array(topics.pluck(:id, :name))
     end
+
+    it 'returns the image_url in the body response' do
+      subject
+      json[:topics].each { |topic| expect(topic).to have_key('image_url') }
+    end
+  end
+
+  context 'When there are topics without image' do
```

Comment:
just to be consistent..
```suggestion
  context 'when there are topics without image' do
```
😅